### ISSUE TITLE
suggestion

### ISSUE THREAD

**Issue by jami223** [SkyTNT/midi-model#16]
It would be interesting if you added text2midi support, so you could generate music the way the person wants

**Comment by jamisson2006** [comment]
Yes, it would be interesting if there was text2midi, so we could generate music of our own taste
